nton/constants.py

```
@@ -28,9 +28,10 @@ class Binaries:
 class Files:
     keys_home = (Path.home() / ".switch" / "prod.keys")
     keys_cwd = Path("./prod.keys").absolute()
-    keys = keys_cwd or keys_home
+    keys_exist = keys_home.is_file() or keys_cwd.is_file()
+    keys = keys_cwd if keys_cwd.is_file() else keys_home
 
 
-if not Files.keys.is_file():
+if not Files.keys_exist:
```

This also should not change. I presume these are squash errors.